tox.ini

```diff
@@ -70,34 +70,34 @@ commands =
     test: coverage run --rcfile=tools/coverage_unit_tests.rc --append --source nimodinst -m py.test bin/nimodinst/nimodinst {posargs} -s
     test: coverage report --rcfile=tools/coverage_unit_tests.rc
     test: coverage html --rcfile=tools/coverage_unit_tests.rc  --directory=bin/htmlcov/unit_tests
-    clean: python --version
-    clean: python -c "import platform; print(platform.architecture())"
+    clean: python3 --version
+    clean: python3 -c "import platform; print(platform.architecture())"
     clean: make clean {posargs}
-    codegen: python --version
-    codegen: python -c "import platform; print(platform.architecture())"
-    codegen: python -m pip install pip --upgrade pip
+    codegen: python3 --version
+    codegen: python3 -c "import platform; print(platform.architecture())"
+    codegen: python3 -m pip install pip --upgrade pip
     codegen: make {posargs} PYTHON_CMD=python
```

Comment:
Shouldn't this be changed to python3 as well?

Reply:
It shouldn't matter since in the venv that is created both python and python3 should be the same thing.

However, I changed it anyway to be consistent.